src/api/app/components/approved_or_declined_review_component.html.haml

```diff
@@ -0,0 +1,19 @@
+- if @key == :accepted_reviews
+  %i.fas.fa-check.text-primary
+- else
+  %i.fas.fa-times.text-danger
+= helpers.image_tag_for(@reviewer, size: 23, custom_class: 'rounded-circle bg-light border border-gray-400')
```

Comment:
When the review is by_group, for example, the avatar of the group is not displayed but the one of the user who reviewed on behalf of the group. 

Is that what we want? Even if the suffix "by X" is also added. See the screenshot:

![Screenshot 2022-08-08 at 13-42-30 Open Build Service](https://user-images.githubusercontent.com/2581944/183411094-435d7d02-ba33-4c17-b430-2120856b2fa9.png)

Reply:
No it's not.

Reply:
I'm working on this, the change will help to simplify the code a bit as well.

Reply:
I already fixed this issue with the avatars. That helped to simplify the code a lot. I added many commits to the PR to make the review easier.

There are more changes to do to how the list looks, but so far the avatars look like this:

![Screenshot 2022-08-09 at 10-04-00 Open Build Service](https://user-images.githubusercontent.com/2581944/183608973-40c14a24-427b-455a-b14c-f44f6c5e1ba3.png)